privacyidea/api/before_after.py

```diff
@@ -186,8 +186,7 @@ def before_request():
                                 get_from_config(SYSCONF.OVERRIDECLIENT))
     # Save the HTTP header in the localproxy object
     g.request_headers = request.headers
-    privacyidea_server = current_app.config.get("PI_AUDIT_SERVERNAME") or \
-                         request.host
+    privacyidea_server = get_privacyidea_node(request.host)
```

Comment:
I would still use the `current_app.config.get("PI_AUDIT_SERVERNAME")` before using `get_privacyidea_node(request.host)`. If someone has configured `PI_AUDIT_SERVERNAME` as well as `PI_NODE`, the `PI_NODE` would have precedence which could be unexpected.